packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -139,7 +139,7 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
 
             const miSdkApiTag = document.createElement('script');
             miSdkApiTag.setAttribute('type', 'text/javascript');
-            miSdkApiTag.setAttribute('src', 'https://app.mapsindoors.com/mapsindoors/js/sdk/4.26.4/mapsindoors-4.26.4.js.gz');
+            miSdkApiTag.setAttribute('src', 'https://app.mapsindoors.com/mapsindoors/js/sdk/CandidateReleases/2024.22.01-rc0/mapsindoors-2024.22.1-rc0.js.gz');
```

Comment:
This points to a candidate release. Is this really what we want to use in the Map Template?